server/kolide/import_config_unmarshaler.go

```
@@ -2,8 +2,10 @@ package kolide
 
 import (
 	"encoding/json"
-	"errors"
 	"strconv"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cast"
```

I somewhat object to this import because it increases the API surface and therefore increases the time spent both on-boarding and maintaining our codebase. 

I also don't like that you've chosen to use a method that ignores the error and just returns an empty string when it fails. IMO that has the potential to lead to subtle bugs which will be difficult to debug in the future.

---

I think that using the `cast` package is fairly reasonable in places where it saves a bunch of lines of code. `cast.ToString` does have different semantics than the existing `toString` method here, so we have to be sure that's intended.

---

I would argue that "saves a bunch of lines of code" == "introduces (len of cast package) lines of code that we are now responsible for maintaining". 

I'm not against imports in general but in this particular case I'd like to see some reasoning about why the import helps and some guarantee that defaulting to `""` will not cause subtle problems.

---

We already use cast in other places for one. The platform attribute is a bit of a special case, it does need to change in that it should be checked in the validation layer for valid platform values, not here, but, if it's nil or missing, then according to the spec, it defaults to all, which in this case is ok